dbt_superset_lineage/push_descriptions.py

```diff
@@ -144,6 +148,9 @@ def merge_columns_info(dataset, tables):
     sst_columns = dataset['columns']
     dbt_columns = tables.get(key, {}).get('columns', {})
 
+    sst_description = dataset['description']
+    dbt_description = tables.get(key, {}).get('description', '')
```

Comment:
This is a super small nitpick but you can maybe consider using `None` instead of an empty string. `None` is the default value.

```suggestion
    dbt_description = tables.get(key, {}).get('description')
```

After that you also need to change the check below.

```python
if dbt_description is None:
```

Reply:
Sure, let's simplify it!